test_runner/fixtures/neon_fixtures.py

```diff
@@ -2938,6 +2938,27 @@ def wait_for_upload(
     )
 
 
+# Does not use `wait_until` for debugging purposes
+def wait_until_tenant_status(
+    pageserver_http: PageserverHttpClient,
+    tenant_id: TenantId,
+    expected_status: str,
```

Comment:
I think state rather than status, not sure about the function name, it was the same before and is using tenant/status endpoint

Reply:
https://github.com/neondatabase/neon/pull/3013 introduces a generic `wait_until` function. Maybe cherry-pick that. Or cherry-pick that and use it to implement `wait_until_tenant_status` as a convenience function.

Reply:
Hm, I don't think https://github.com/neondatabase/neon/pull/3013 introduced that, at least there's already `utils.py` that has `wait_until` with a better signature even.
There are with that function (and I presume, the new one too):
* it prints the function as `lambda ...` which is not very informative
* `log.error` and others does not seem to work when placed inside the lambda

That's why I've created a separate function that's easier to debug, if needed.
I think I'll stick to that for now and won't use that lambda-based function.